backend/requirements.txt

```diff
@@ -21,7 +21,10 @@ gensim==4.3.2
 # spicy==1.12.0  # this important, 1.13.0 will give error
 scikit-learn==1.2.2 # this important, because the gb_model_v1.pkl model is trained with this version
 PyPDF2==3.0.1
-xgboost==2.0.3
+xgboost==1.4.0
+google-cloud-aiplatform>=0.8.0
+google-cloud-texttospeech
+
```

Comment:
i believe this also need
vertexai==1.48.0

Reply:
done